apps/sync/test/unit/fixtures/fxsyncwebcrypto.js

```diff
@@ -0,0 +1,62 @@
+/* exported FxSyncWebCryptoFixture */
+
+'use strict';
+
+var FxSyncWebCryptoFixture = (function fxsync_webcrypto_fixture() {
+  // NB: This data is from an FxSync account, freshly created for this purpose,
+  // and then synced once using FxDesktop with a fresh empty Firefox profile:
+
+  function hex2ba(hexStr) {
+    var numBytes = hexStr.length / 2;
+    var byteArray = new Uint8Array(numBytes);
+    for (var i = 0; i < numBytes; i++) {
+      byteArray[i] = parseInt(hexStr.substr(i * 2, 2), 16);
+    }
+    return byteArray;
+  }
+
+  return {
+    kB: '85c4f8c1d8e3e2186824c127af786891dd03c6e05b1b45f28f7181211bf2affb',
+    hkdf: {
+      kB: hex2ba(
+          '85c4f8c1d8e3e2186824c127af786891dd03c6e05b1b45f28f7181211bf2affb'),
+      infoStr: hex2ba(
+          '6964656e746974792e6d6f7a696c6c612e636f6d2f7069636c2f76312f6f6c647' +
+          '3796e63'), // 'identity.mozilla.com/picl/v1/oldsync'
+      outputHex:
+          'd63eb2610f1e5edc21b37fed22b93b0f9ea379840fb5c3c5eb95991b9b3bfab69' +
+          'cbabfda0762bdf0b27431d81960c8cf5dab7d482901213a5a1b003eb0081ed0'
+    },
+    cryptoKeys: {
+      ciphertext: 'PP5yNUYwJJoLcsL5o85i6RZfvanYDrwtChDD/LdKTZ8JOLubZ9DyRv3HMe' +
```

Comment:
Can we use es6 template strings for multiline strings here and in the rest of the patch, please?